src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,9 +7,7 @@
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
+    public static final Prefix PREFIX_IMPORTANCE = new Prefix("i/");
```

Comment:
since our format for the argument prefix is `/cmd Argument`, should this be `/i` instead?